galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -209,3 +209,16 @@ messages:
     args:
       - name: deployment
         type: string
+
+  - name: "PortNameIsNotUnderNamingConvention"
+    code: IST0118
+    level: Warning
+    description: "Port name is not under naming convention. Protocol detection is applied to the port."
```

Comment:
Why is this a warning? The whole point of protocol detection was that a user does not need to care about the port name, but now we are going to spit out 1000s of messages about the port name whne they run this?

Reply:
I think make the cmd optional and set the level Warning would be good. If we cannot make the cmd as optional, I will change this to Info level.

Reply:
There is an open issue to make the cmd optional https://github.com/istio/istio/issues/17443. The milestone is 1.5. I think we could keep this message level warning.

Reply:
Or we can turn this to warning when  #17443 is done.